src/india_api/internal/service/__init__.py

```diff
@@ -1,3 +1,4 @@
-from .server import server, get_db_client
+from .server import server
+from .database_client import get_db_client
```

Comment:
Unused import now this is split and not in the array..?

Reply:
to be honest, im not quite sure what this file does, do perhaps ill put it back to what it was